src/EditorFeatures/Core/Implementation/InlineRename/CommandHandlers/RenameCommandHandler_RenameHandler.cs

```diff
@@ -80,6 +73,25 @@ private void ExecuteRenameWorker(RenameCommandArgs args, CancellationToken cance
                 return;
```

Comment:
Thinking about what we want here.  Commit and then show the error? Commit and do nothing?  Do nothing in general?